style.css

```diff
@@ -29,6 +38,11 @@ body {
   --slider-height: calc(var(--switch-height) - var(--switch-padding));
   --slider-on: calc(var(--switch-height) - var(--switch-padding));
 }
+
+.power-switch {
+  margin: 3px;
+}
+
```

Comment:
these changes will only bloat the css file and slow down loading.

Reply:
I just removed it from the html and added it to the css, but we can remove it.

Reply:
> I just removed it from the html and added it to the css, but we can remove it.

it just makes sense to keep it inline as this way it loads faster as inline CSS is directly available

Reply:
If you remove this margin, it will put the item on the red line

Reply:
> I decided to remove the css in the html to make your html file more clean, since using styles.css in the project.

but inline CSS allows for faster load speeds

Reply:
> In your project, impact is not considered, so overloading doesn't make sense to me. There are other techniques to minify your css.

I realise that but I don't see this to be necessary?

Reply:
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Organizing